docker/docker-compose.yml

```diff
@@ -1,184 +1,194 @@
 version: "3"
```

Comment:
@BugDiver - is this only formatting change?

Reply:
It has formatting changes, along with the actual changes. We have added a new service for France's data producer.

Reply:
sure. looks fine